apps/api/src/app/events/services/sms-service/handlers/burstsms.handler.ts

```diff
@@ -0,0 +1,18 @@
+import { ChannelTypeEnum } from '@novu/shared';
```

Comment:
I forgot to ask to rename this file to `burst-sms.handler.ts`. Apologies for that! 🙏🏻

Reply:
No worries. Sorry I probably should have caught that one myself when doing the changes yesterday